### PR TITLE
Climate 101 to fix #1656

### LINKED ISSUE #1656: New tutorial Climate 101 planned
I just would like to make everybody aware that we are starting to work on a new Galaxy tutorial "Climate 101". The aim is to give an introduction to Climate data.

Let me know if you are interested to participate.

### PR DESCRIPTION
**Pull request by annefou**
- drafted tutorial but still needs a lot of work.

### CODE REVIEW

File: topics/climate/tutorials/climate-101/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,399 @@
...
+
+## Past, present and future climate?
+
+
+When we talk about climate data, the type of data can vary significantly. We have very little actual observations at the scale of climate and usually not covering a large area. In addition to observations, we can make use of:
+- Re-analyses where observations and numerical modelling are combined together. 
+- Climate models.
+
+Observations and re-analyses provide information about the past and current climate while climate models can provide past, current and future climate information.
+When it comes to future climate, we usually need to make some assumptions (such as how much CO2 emissions, etc.) and make different scenarios e.g. we run climate models using different assumptions and look at future trends under each of these scenarios: this is what we call **climate projections**. Climate projections will be discussed in a separate Galaxy tutorial.
```
Comment (bgruening) [inline topics/climate/tutorials/climate-101/tutorial.md:391]: "and make different scenarios" sounds strange to me ... simulate different scenarios maybe?

File: topics/climate/tutorials/climate-101/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,399 @@
...
+The figure below shows a woman walking her dog and we can use it to make an analogy to illustrate the difference between weather and climate. 
+if you focus your attention on the dog, you can see that it is all over the place, sometimes upwards, sometimes downwards: this can represent the weather and its 
+variability. The dog (weather) is not following a fully random pattern and varies around a main direction (trend) that is given by the woman: the woman is representing
+the climate and gives us an indication of where both the woman and dog are likely to be in the future.
+  
+ ![Illustrate the difference beteen weather and climate](../../images/weather_versus_climate.png "Weather versus Climate")
+
+*Source: [Animated short on statistics](https://youtu.be/e0vj-0imOLw) from Norwegian infotainment program Siffer. Produced by TeddyTV for NRK. Animation by Ole Christoffer Haga*
+
+You can also watch this [Video](https://youtu.be/e0vj-0imOLw) to get an animated illustration of the difference between climate and weather.
```
Comment (bgruening) [inline topics/climate/tutorials/climate-101/tutorial.md:115]: you can embed videos into the training with:

`<div class="embed-responsive embed-responsive-16by9"><iframe src="https://player.vimeo.com/video/120901536?portrait=0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe></div>` as seen here: https://raw.githubusercontent.com/galaxyproject/training-material/master/topics/introduction/tutorials/galaxy-intro-ngs-data-managment/tutorial.md

File: topics/climate/tutorials/climate-101/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,399 @@
+---
+layout: tutorial_hands_on
+title: Getting your hands-on climate data
+zenodo_link: ''
```
Comment (bgruening) [inline topics/climate/tutorials/climate-101/tutorial.md:4]: https://doi.org/10.5281/zenodo.3776499

**Comment by annefou** [comment]
Thanks @bgruening for all your tips and recommendations. I think I have addressed all of them.
Many thanks.

File: topics/climate/tutorials/climate-101/tutorial.md [hunk 4]
```diff
@@ -354,29 +354,37 @@ At the moment, there are [54 ECVs](https://gcos.wmo.int/en/essential-climate-var
 
 > ### {% icon hands_on %} Hands-on: Essential Climate Variables
 >
+> We will look at the [Water Vapor Essential Climate Variable ](https://gcos.wmo.int/en/essential-climate-variables/surface-vapour/): 
+> *The humidity of air near the surface of the Earth affects the comfort and health of humans, livestock and wildlife, the swarming behaviour of insects and the occurrence of plant disease. The humidity near the surface affects evaporation and the strength of the hydrological and energy cycles. Evaporation from the surface of the earth is the source of water in the atmosphere and so is responsible for important feedbacks in the climate system due to clouds and radiation.*
```
Comment (bgruening) [inline topics/climate/tutorials/climate-101/tutorial.md:358]: Is there a difference between ` The humidity near the surface` and ` The humidity of air near the surface` ?

File: topics/climate/tutorials/climate-101/tutorial.md [hunk 5]
```diff
@@ -397,6 +405,8 @@ When it comes to future climate, we usually need to make some assumptions (such
 
 # Conclusion
 
+This tutorial taught you some of the basics of climate data analysis, explaining the difference between weather and climate as well as the existance of Essential Climate Variables to help you choosing variables that can help you in your analysis. In future Galaxy tutorials, we will go into details on where to get Climate data and how to make statistical analysis to get an insight on the climate change.
```
Comment (bgruening) [inline topics/climate/tutorials/climate-101/tutorial.md:408]: this is now semi-redundant with the sentence below.

File: topics/climate/tutorials/climate-101/tutorial.md (hunk 4 above)
Comment (annefou) [reply topics/climate/tutorials/climate-101/tutorial.md:358]: Well, picky climate scientists would argue that using the term "The humidity near the surface" is imprecise so we better use "the humidity of air near the surface"...

Comment (bgruening) [reply topics/climate/tutorials/climate-101/tutorial.md]: I was just confused as you use both term, in two nearby sentences ... maybe you can stick to one?

File: topics/climate/tutorials/climate-101/tutorial.md (hunk 5 above)
Comment (annefou) [reply topics/climate/tutorials/climate-101/tutorial.md]: Very redundant. I forgot we had already a conclusion! I removed what I just added...

**Pull request merged**